src/main/java/micdoodle8/mods/galacticraft/planets/asteroids/items/ItemGrappleHook.java

```diff
@@ -14,17 +14,16 @@
 import net.minecraft.init.Items;
 import net.minecraft.init.SoundEvents;
 import net.minecraft.item.*;
-import net.minecraft.util.ActionResult;
-import net.minecraft.util.EnumActionResult;
-import net.minecraft.util.EnumHand;
-import net.minecraft.util.SoundCategory;
+import net.minecraft.util.*;
 import net.minecraft.util.text.TextComponentString;
 import net.minecraft.world.World;
 import net.minecraftforge.fml.relauncher.Side;
 import net.minecraftforge.fml.relauncher.SideOnly;
+import net.minecraftforge.oredict.OreDictionary;
 
 public class ItemGrappleHook extends ItemBow implements ISortableItem
 {
+    private static final NonNullList<ItemStack> STRING_ENTRIES = OreDictionary.getOres("string");
```

Comment:
Isn't this static code calling `OreDictionary.getOres()` too early?    `ItemGrappleHook` will be called first some time when Galacticraft is registering its items, and other mods may not yet have registered their items.   (Probably only mods with names beginning A-F have registered items already at this point!)

I think this probably needs to be initialised on first reference from `onPlayerStoppedUsing()`.